src/util/color/predefinedcolorpalettes.cpp

```diff
@@ -159,6 +159,85 @@ constexpr mixxx::RgbColor kMixxxKeyColor10(0x3D8AFD);
 constexpr mixxx::RgbColor kMixxxKeyColor11(0xAC64FE);
 constexpr mixxx::RgbColor kMixxxKeyColor12(0xFD3FEA);
 
+// Traktor Key Color Palette
+// NOTE: C Major is 1d in Open Key Notation
+
+constexpr mixxx::RgbColor kTraktorKeyColor1(0xB960A2);
+constexpr mixxx::RgbColor kTraktorKeyColor2(0x8269AB);
+constexpr mixxx::RgbColor kTraktorKeyColor3(0x527FC0);
+constexpr mixxx::RgbColor kTraktorKeyColor4(0x3CC0EF);
+constexpr mixxx::RgbColor kTraktorKeyColor5(0x5BC1CE);
+constexpr mixxx::RgbColor kTraktorKeyColor6(0x4CB686);
+constexpr mixxx::RgbColor kTraktorKeyColor7(0x73B629);
+constexpr mixxx::RgbColor kTraktorKeyColor8(0x9FC516);
+constexpr mixxx::RgbColor kTraktorKeyColor9(0xFDD615);
+constexpr mixxx::RgbColor kTraktorKeyColor10(0xF28B2E);
+constexpr mixxx::RgbColor kTraktorKeyColor11(0xEC6637);
+constexpr mixxx::RgbColor kTraktorKeyColor12(0xE84C4D);
+
+// Mixed In Key Key Color Palette
+// NOTE: C Major is 8B in Camelot Notation
+
+constexpr mixxx::RgbColor kMIKKeyColor1(0xF17EDB);
+constexpr mixxx::RgbColor kMIKKeyColor2(0xD18BFD);
+constexpr mixxx::RgbColor kMIKKeyColor3(0x9EB4FD);
+constexpr mixxx::RgbColor kMIKKeyColor4(0x4DD3F8);
+constexpr mixxx::RgbColor kMIKKeyColor5(0x01EAEC);
+constexpr mixxx::RgbColor kMIKKeyColor6(0x00EECB);
+constexpr mixxx::RgbColor kMIKKeyColor7(0x20EF7F);
+constexpr mixxx::RgbColor kMIKKeyColor8(0x7FF448);
+constexpr mixxx::RgbColor kMIKKeyColor9(0xE0CA6D);
+constexpr mixxx::RgbColor kMIKKeyColor10(0xFDA078);
+constexpr mixxx::RgbColor kMIKKeyColor11(0xFF8693);
+constexpr mixxx::RgbColor kMIKKeyColor12(0xFD7EB3);
+
+// Accessible Color Palettes
```

Comment:
An additional sentence about the trick you have used and the goal you achieved would be nice for later reference